pkg/gator/validate/validate_test.go

```diff
@@ -0,0 +1,107 @@
+package validate
+
+import (
+	"testing"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/gator/fixtures"
+	y3 "gopkg.in/yaml.v3"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+func TestValidate(t *testing.T) {
+	tcs := []struct {
+		name   string
+		inputs []string
+		want   int
+		err    bool
```

Comment:
Use errors  instead of bool (`wantErr`) and check against `errors.Is` if reasonable

Reply:
The tests don't currently have any of these cases, but I've changed it to `error` from `bool` and included a comment about `errors.Is()`.  Will use that if it is appropriate given new test cases.